base_layer/wallet_ffi/python/tari_wallet/config_reader.py

```
@@ -0,0 +1,381 @@
+            Parsed configuration dictionary
+        """
+        config = {}
+        current_section = None
+        current_array_key = None
+        current_array_items = []
+        in_multiline_array = False
+        
+        lines = content.split('\n')
+        
+        for line in lines:
+            original_line = line
```

Flake8 (7.2.0)

[error] local variable 'original_line' is assigned to but never used

(F841)